### PR TITLE
Feature/profile

### PR DESCRIPTION
**Pull request by jgiwinski**
## Is this a fix or a feature?
Feature 

## What is the change?
Build and style profile page.
Build Service component for user profile and create new account page. 

## What does it fix?
NA

## Where should the reviewer start?
Profile.vue

## How should this be tested?
NA

### CODE REVIEW

File: src/_mixins.scss [hunk 1]
```diff
@@ -17,4 +17,9 @@
   transform: scale(1.1);
   transition: transform 300ms;
   transition: all 0.2s ease-in-out;
+}
+
+@mixin flex-row {
```
Comment (mattumland) [inline src/_mixins.scss:22]: Love the extra mixin. Great job creating a reusable tool

File: src/components/Profile.vue [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+    props: {
+        currentUser: {type: Object}
+    }, 
+    components: {
+        Service
+    }, 
+    data() {
+        return {
+            subscriptionList: [
+                {id: 0, sub: "NutFlex"},
```
Comment (mattumland) [inline src/components/Profile.vue:35]: We'll need to pass this down as a prop once we're able to get it from the BE.

**Review by mattumland — APPROVED** [review]

File: src/components/Profile.vue [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+        <div class="welcome-container">
+            <h1>Welcome, {{ `Nick Cage`|| currentUser.first_name }}</h1>
+            <h2 class="email">Email: {{ 'nick.cage@gmail.com' || currentUser.email }}</h2>
+        </div>
+        <div class="lang-reg-container">
+            <h2>Language: {{ 'English' || currentUser.language }} | </h2>
+            <h2>| Region: {{ 'United States' || currentUser.region }}</h2>
+        </div>
+        <h2>YOUR SERVICES:</h2>
+        <article class="subscriptions-container">
```
Comment (Jacksonmcguire) [inline src/components/Profile.vue:12]: This is looking great

**Review by Jacksonmcguire — APPROVED** [review]
All looks great, think we've got all of the views set now.

**Pull request merged**